dataactbroker/handlers/fileHandler.py

```diff
@@ -1793,20 +1797,25 @@ def serialize_submission(submission):
     files = get_submission_files(jobs)
     status = get_submission_status(submission, jobs)
     certified_on = get_lastest_certified_date(submission)
+    window_end = get_window_end(submission)
+    time_period = get_time_period(submission)
     agency_name = submission.cgac_agency_name if submission.cgac_agency_name else submission.frec_agency_name
     return {
-        "submission_id": submission.submission_id,
-        "last_modified": str(submission.updated_at),
-        "status": status,
-        "agency": agency_name if agency_name else 'N/A',
-        "files": files,
+        'submission_id': submission.submission_id,
+        'last_modified': str(submission.updated_at),
+        'status': status,
+        'agency': agency_name if agency_name else 'N/A',
+        'files': files,
         # @todo why are these a different format?
-        "reporting_start_date": str(submission.reporting_start_date) if submission.reporting_start_date else None,
-        "reporting_end_date": str(submission.reporting_end_date) if submission.reporting_end_date else None,
-        "user": {"user_id": submission.user_id, "name": submission.name if submission.name else "No User"},
-        "certifying_user": submission.certifying_user_name if submission.certifying_user_name else "",
+        'reporting_start_date': str(submission.reporting_start_date) if submission.reporting_start_date else None,
+        'reporting_end_date': str(submission.reporting_end_date) if submission.reporting_end_date else None,
+        'user': {'user_id': submission.user_id, 'name': submission.name if submission.name else 'No User'},
+        'certifying_user': submission.certifying_user_name if submission.certifying_user_name else '',
         'publish_status': PUBLISH_STATUS_DICT_ID[submission.publish_status_id],
-        "certified_on": str(certified_on) if certified_on else ""
+        'certified_on': str(certified_on) if certified_on else '',
+        'quarterly_submission': submission.is_quarter_format,
```

Comment:
Would it be easier for frontend if we sent the actual text of "Quarterly" or "Monthly" or is this more useful to them? I'm not sure what else they would do with this information except display it.

Reply:
I could do either here as well. I just thought having it as a boolean would be more direct with what we have in the database and they could use the boolean for whatever logic they have on the frontend.